arch/x86/mmu.c

```diff
@@ -442,7 +442,7 @@ static void  *walk_paging_struct(void *addr, void *table_base,
 				    (uint64_t) sub_table_addr | table_present);
 		} else {
 			/* Get address of the sub-table */
-			sub_table_addr = (void *)(table_entry & IA32E_REF_MASK);
+			sub_table_addr = (void *)(table_entry & CPU_PAGE_MASK);
```

Comment:
This file is still used for EPT. (Of course it is also used in paging_init of hypervisor).
In such case the IA32E_REF_MASK is better.